xil_carma_messenger/.env

```diff
@@ -1,7 +1,7 @@
 # This env file is built into the carma config image and contains only image version
 # variables to pass into carma config docker compose.
 # NOTE: Editing this file locally without rebuilding carma config image will have no impact
-DOCKER_TAG=develop
-DOCKER_ORG=usdotfhwastoldev
-OPS_DOCKER_TAG=develop
+DOCKER_TAG=prowler
+DOCKER_ORG=usdotfhwastolcandidate
+OPS_DOCKER_TAG=7.8.1
```

Comment:
This variable is not being used. is that intentional?
It seems to hardcode to 7.7.0 (php and v2xhub), in that case, let's either remove it or match to 7.7.0 because it might create confusion

Reply:
Good catch. I think it should be ok to remove the tag for v2x hub here since we intend this config to use the hardcoded version

Reply:
commented out OPS_DOCKER_TAG variable